string capability
string bond_id
---
